components/UserOverview/UserOverview.tsx

```diff
@@ -52,12 +52,17 @@ const useStyles = makeStyles((theme) => ({
     flexDirection: "column",
     justifyContent: "center",
     alignItems: "stretch",
-    padding: theme.spacing(0.5),
+    padding: "0px",
     "& > *": {
       margin: theme.spacing(0.5),
     },
     "& > :not(:first-child)": {
       marginLeft: theme.spacing(0.5),
+      marginTop: theme.spacing(1.5),
+    },
+    "& > button": {
+      fontWeight: "bold",
+      padding: theme.spacing(1.6),
```

Comment:
Why 1.6? If it's to pixel-match figma I'd probably leave it as 1.5 for consistency (we tend to try to have all uses of spacing in 0.5 increments)